src/internal/utils.js

```diff
@@ -20,19 +20,20 @@ export function check(value, predicate, error) {
 }
 
 export const is = {
-  undef     : v => v === null || v === undefined,
-  notUndef  : v => v !== null && v !== undefined,
-  func      : f => typeof f === 'function',
-  number    : n => typeof n === 'number',
-  array     : Array.isArray,
-  promise   : p => p && is.func(p.then),
-  iterator  : it => it && is.func(it.next) && is.func(it.throw),
-  task      : t => t && t[TASK],
-  observable: ob => ob && is.func(ob.subscribe),
-  buffer    : buf => buf && is.func(buf.isEmpty) && is.func(buf.take) && is.func(buf.put),
-  pattern   : pat => pat && ((typeof pat === 'string') || (typeof pat === 'symbol') || is.func(pat) || is.array(pat)),
-  channel   : ch => ch && is.func(ch.take) && is.func(ch.close),
-  helper    : it => it && it[HELPER]
+  undef       : v => v === null || v === undefined,
+  notUndef    : v => v !== null && v !== undefined,
+  func        : f => typeof f === 'function',
+  number      : n => typeof n === 'number',
+  array       : Array.isArray,
+  promise     : p => p && is.func(p.then),
+  iterator    : it => it && is.func(it.next) && is.func(it.throw),
+  task        : t => t && t[TASK],
+  observable  : ob => ob && is.func(ob.subscribe),
+  buffer      : buf => buf && is.func(buf.isEmpty) && is.func(buf.take) && is.func(buf.put),
+  pattern     : pat => pat && ((typeof pat === 'string') || (typeof pat === 'symbol') || is.func(pat) || is.array(pat)),
+  channel     : ch => ch && is.func(ch.take) && is.func(ch.close),
+  helper      : it => it && it[HELPER],
+  stringableFn: f => is.func(f) && f.hasOwnProperty('toString')
```

Comment:
Hello @Andarist! Thank you for taking the time to review this for me!

I have the other changes in place but I am confused by what you mean here. Could you elaborate? Are you saying it would choke if someone defined a custom `hasOwnProperty` method the input pattern so we should store it as a util?